cachito/web/api_v1.py

```diff
@@ -406,6 +410,14 @@ def patch_request(request_id):
                 "Failed to delete the temporary files at %s", bundle_dir
             )
 
+    if delete_logs:
+        request_log_dir = flask.current_app.config["CACHITO_REQUEST_FILE_LOGS_DIR"]
+        path_to_file = os.path.join(request_log_dir, f"{request_id}.log")
+        try:
+            os.path.remove(path_to_file)
```

Comment:
`os.path.remove` does not exist. Did you mean `os.remove(path_to_file)`?

Reply:
Yes good catch! Thanks